### DISCUSSION TITLE
Difference between the LDSRG(2) and LDRSG(2)_QC methods

### DISCUSSION THREAD

**Discussion by vvp-nsk** [evangelistalab/forte#390] [Q&A]
Hej!

I am puzzled a bit on what is the difference between LDSRG2_QC and LDSRG2?  Is LDSRG2_QC equal to LDSRG2 when DSRG_RSC_NCOMM=2? 

Sorry for bothering and thank you in advance!

With best regards, 
Victor

**Comment by lcyyork** [comment]
You are correct. The results should be the same. The difference  is that qc is more memory friendly for conventional 4-index integrals.
